packages/ti_abusech/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.6.1"
+  changes:
+    - description: Enable Agentless deployment.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/13173
```

```suggestion
- version: "2.7.0"
  changes:
    - description: Enable Agentless deployment.
      type: enhancement
      link: https://github.com/elastic/integrations/pull/13173
```
Since its an enhancement.

---

fixed